x-pack/plugins/ml/server/routes/results_service.ts

```diff
@@ -352,4 +354,39 @@ export function resultsServiceRoutes({ router, routeGuard }: RouteInitialization
       }
     })
   );
+
+  /**
+   * @apiGroup ResultsService
+   *
+   * @api {post} /api/ml/results/datafeed_results_chart Get datafeed results chart data
+   * @apiName GetDatafeedResultsChartData
+   * @apiDescription Returns datafeed results chart data
+   *
+   * @apiSchema (body) getDatafeedResultsChartDataSchema
+   */
+  router.post(
+    {
+      path: '/api/ml/results/datafeed_results_chart',
+      validate: {
+        body: getDatafeedResultsChartDataSchema,
+      },
+      options: {
+        tags: ['access:ml:canGetJobs'],
+      },
+    },
+    routeGuard.fullLicenseAPIGuard(async ({ client, mlClient, request, response }) => {
+      try {
+        const { getDatafeedResultsChartData } = resultsServiceProvider(mlClient, client);
+        const resp = await getDatafeedResultsChartData(
+          request.body as DatafeedResultsChartDataParams
```

Comment:
this cast to `DatafeedResultsChartDataParams` is unnecessary

Reply:
Removed in 1df1594527f9d947920775b41019c80641ca2777